README.md

```diff
@@ -151,11 +153,16 @@ Some simple sample data is injected into the MongoDB instance when created. If m
 
 The following steps are intended to help get a new user up and running the JPO CV Manager in their own environment.
 
-1.  Follow the Requirements and Limitations section and make sure all requirements are met.
-2.  Create a copy of the sample.env named ".env" and refer to the Environmental variables section below for more information on each variable.
+1. Follow the Requirements and Limitations section and make sure all requirements are met.
+2. Run the following command to initialize submodules:
+    ```sh
+    git submodule update --init --recursive
+    ```
+3. Create a copy of the sample.env named ".env" and refer to the Environmental variables section below for more information on each variable.
```

Comment:
The creation of the `.env` file in the `/jpo-utils` folder is no longer required if we opt to use the CDOT fork of `jpo-utils`. We would want to eventually swap over to the USDOT one during the release but figured I'd let you know.

Reply:
@payneBrandon Should we point to CDOT jpo-utils or USDOT jpo-utils in this PR?

Reply:
Is that one getting merged into usdot with this release?

Reply:
Yes it should be

Reply:
Addressed here: https://github.com/CDOT-CV/jpo-cvmanager/pull/194